### PR TITLE
Runtime test: Logs gathering fixes

### PR DESCRIPTION
**Pull request by eloycoto**
Three separate commits: 

1) Use BeforeAll/AfterAll in Kafka runtime. 
2) Fix a log message in the test/helpers/cilium.go
3) Fix the issue with the `BeforeEach` and logs gathering. 

Regards.

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4511)

### CODE REVIEW

**Comment by eloycoto** [comment]
test-me-please

**Comment by aanm** [comment]
@eloycoto does this needs to be backported?

**Comment by eloycoto** [comment]
@aanm monitor is not part of v1.0 branch.

File: test/runtime/kafka.go [hunk 1]
```diff
@@ -110,24 +102,32 @@ var _ = Describe("RuntimeValidatedKafka", func() {
 		return err
 	}
 
-	BeforeEach(func() {
-		once.Do(initialize)
+	BeforeAll(func() {
+		logger = log.WithFields(logrus.Fields{"testName": "RuntimeKafka"})
```
Comment (ianvernon) [inline test/runtime/kafka.go]: Test name does not match the actual test name. This should be made into a variable and used in both the `Describe` and this logger instantiation accordingly.

**Review by ianvernon — APPROVED** [review]
One small nit. Can be fixed without need for re-review. The rest LGTM.

**Comment by eloycoto** [comment]
test-me-please

**Pull request merged**